### PR TITLE
Allow schools to have dashboard messages

### PR DESCRIPTION
**Pull request by urbanwide**

### CODE REVIEW

File: app/views/schools/meters/index.html.erb [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <h1><%= t('schools.meters.index.title') %></h1>
 
 <% if current_user.admin? %>
-<div class="mb-2 alert alert-secondary">
+  <% if can? :manage, DashboardMessage %>
+    <%= render 'admin/shared/dashboard_message', messageable: @school %>
+  <% end %>
+<div class="alert alert-secondary mb-2">
   <%= link_to t('schools.meters.index.school_downloads'), school_downloads_path(@school), class: 'btn' %>
```
Comment (urbanwide) [inline app/views/schools/meters/index.html.erb:8]: This looks a bit sus. Think the link_to should be inside the following?
<% if can?(:validate_meters, @school) && @school.meters_with_readings.any? %>
<% end %>

File: app/views/schools/meters/index.html.erb [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
   <% if can? :manage_solar_feed_configuration, School %>
   <% end %>
```
Comment (urbanwide) [inline app/views/schools/meters/index.html.erb:10]: This looks a bit sus. Think the link_to perhaps should be inside this if block?

**Review by iant — APPROVED** [review]
:+1: all reviewed and looks good to me @urbanwide 

@ldodds could you have a quick look at the comment on [app/views/schools/meters/index.html.erb](https://github.com/Energy-Sparks/energy-sparks/pull/2293/files#diff-ca3498b619fe616b0f4809456cd29eb6735532d8b599c29f25fbb92649177ff8) (if the manage_solar_feed_configuration should be removed or the link moved)

**Comment by urbanwide** [comment]
Thanks @iant, I'll merge this down and make a note of the above in a ticket.

**Pull request merged**